server/index.js

```diff
@@ -5,13 +5,24 @@ const port = parseInt(process.env.PORT, 10) || 3000;
 const dev = process.env.NODE_ENV !== 'production';
 const app = next({ dev });
 const handle = app.getRequestHandler();
+const multer = require('multer');
+const path = require('path');
 
 const content = require('./content');
 const contentType = require('./content-type');
 
+const storage = multer.diskStorage({
+  destination: './static',
+  filename(req, file, cb) {
```

Comment:
1. looks like filename is not defined
2. also, we should have the logic to deal with files name collisions. Use case: somebody uploads the file and there is another file with the same name in the folder. Typically, files are grouped by the `uploads/yyyy-mm-dd/` folder pattern and in addition if the file name is already exists, simply append suffix like `-1`, `-2` and etc. Folder pattern adds the creation date to the image url which sort of good for SEO, but I might be wrong about it